common/algorithm-terms.html

```diff
@@ -20,6 +40,44 @@
     used internally to help determine if object embedding is appropriate,</span>
     the <a>explicit inclusion flag</a>,
     and the <a>omit default flag</a>.</dd>
+  <dt><dfn data-cite="JSON-LD11-API#dfn-iri-compacting">iri compacting</dfn></dt><dd class="algorithm">
```

Comment:
Use uppercase "IRI" instead of lowercase "iri" for both "IRI compacting" and "IRI expanding".